source/_integrations/srp_energy.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: "SRP Energy"
+description: "How to integrate SRP Energy within Home Assistant."
+ha_category:
+  - Energy
+ha_release: 0.13
+ha_iot_class: Cloud Polling
+---
+
+The `srp_energy` component shows information from Srp hourly energy usage report for their customers.
+
+You need a Username, Password, and AccountId which you can create at [Srp](https://www.srpnet.com).
+
+## Configuration
+
+To add Srp Energy to your installation, add the following to your `configuration.yaml` file:
```

Comment:
I don't think this is accurate? The integration uses a config flow, so you shouldn't need to add anything to configuration.yaml?